src/test/kotlin/com/atlassian/performance/tools/lib/SshPostgresClient.kt

```diff
@@ -28,4 +30,11 @@ class SshPostgresClient(
         newValue = "\\$character"
     )
 
-}
\ No newline at end of file
+    override fun runSql(
+        ssh: SshConnection,
+        sql: File
+    ): SshConnection.SshResult {
+        ssh.upload(sql, sql.name)
+        return ssh.execute("$psql -f ${sql.name}")
```

Comment:
It will be better to remove the file afterwards. But I don't see an issue to leave the file there for our AWS based testing though, the DB instance is not going to persist.

Reply:
Nice find
Even though the DB will perish and the DB is not password-protected anyway, we still should remove this attack vector. If in the future we'll care more about security and protect the DB more strongly, we'll have one less thing to worry about.